syntax = "proto2";
package Caster.protos;


import "Caster/protos/input_reader.proto";
import "Caster/protos/model.proto";
import "Caster/protos/train.proto";
import "Caster/protos/eval.proto";
// import "Caster/protos/eval.proto";


message TrainEvalConfig {
    optional Model model = 1;
    optional TrainConfig train_config = 2;
    repeated InputReader train_input_reader = 3;
    optional EvalConfig eval_config = 4;
    optional InputReader eval_input_reader = 5;
}